geo-types/src/geometry/polygon.rs

```diff
@@ -244,6 +247,61 @@ impl<T: CoordNum> Polygon<T> {
         self.exterior.close();
     }
 
+    /// Execute the provided closure `f`, which is provided with a mutable
+    /// reference to the exterior `LineString` ring.
+    ///
+    /// After the closure executes, the exterior `LineString` [will be closed].
+    ///
+    /// # Examples
+    ///
+    /// ```
```

Comment:
I've been typically adding `rust, no_run`, because docs.rs builder doesn't have GDAL pre-installed in the environment.